content/getting-started/accounts.md

```diff
@@ -0,0 +1,15 @@
+---
+menu:
+  main:
+    parent: getting-started
+title: Accounts
+weight: -100
+---
+
+## Setting up your account
+
+You will need a Cloud Foundry account before continuing. If you don't already have an account, please [create an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
```

Comment:
Absolutely not. The process right now should be to ask someone who already has access to invite you using that page, even for 18F'ers.